.gitignore

```diff
@@ -6,4 +6,4 @@ node_modules/
 /tmp/
 /dist-assets/
 .eslintcache
-.env
+.env
```

Comment:
Please remove this change by restoring the exact file or by adding a newline at the end.

Reply:
Hello, this is actually the original file, but I submitted it once in order to ignore a certain file. The last time I submitted this file, it was already the original file in the warehouse
In general, please ignore the modification of this file, it does not actually make any changes, it is just for correction, I submitted this file by mistake earlier

Reply:
If you still think it has to be uncommittable, then I re-fork the repository and reapply a new pull request

@tophf